impeller/compiler/reflector.cc

```diff
@@ -402,8 +403,36 @@ std::shared_ptr<fml::Mapping> Reflector::InflateTemplate(
       inflated_template->size(), [inflated_template](auto, auto) {});
 }
 
+std::vector<size_t> Reflector::ComputeOffsets(
+    const spirv_cross::SmallVector<spirv_cross::Resource>& resources) const {
+  std::vector<size_t> offsets(resources.size(), 0);
+  if (resources.size() == 0) {
+    return offsets;
+  }
+  for (const auto& resource : resources) {
+    const auto type = compiler_->get_type(resource.type_id);
+    auto location = compiler_->get_decoration(
+        resource.id, spv::Decoration::DecorationLocation);
+    // Malformed shader, will be caught later on.
```

Comment:
Why not have some kind of VALIDATION_LOG here?

Reply:
So this happens with invalid inputs, there is already a validation log that fires and exit that happens, so we'd either need to perform all of that work twice (since this code only runs on vertex shaders), or we do what I did here and let the later stage catch it.